packages/build/tests/time/snapshots/tests.js.md

```diff
@@ -34,7 +35,8 @@ Generated by [AVA](https://avajs.dev).
     buildbot.build.stage.duration:0|d|#stage:run_plugins,parent:run_netlify_build␊
     buildbot.build.stage.duration:0|d|#stage:start_plugins,parent:run_netlify_build␊
     buildbot.build.stage.duration:0|d|#stage:system,parent:run_netlify_build_per_type␊
-    buildbot.build.stage.duration:0|d|#stage:user,parent:run_netlify_build_per_type`
+    buildbot.build.stage.duration:0|d|#stage:user,parent:run_netlify_build_per_type␊
+    buildbot.build.stage.error:0|c|#stage:system,parent:run_netlify_build`
```

Comment:
Is this expected?

Reply:
This one I'm unsure of, it kept bothering me about it :/ honestly I don't fully know why

Reply:
We should try to get to the bottom of it before merging to make sure we're not introducing a regression.

Reply:
100%